### ISSUE TITLE
Time processor throws an ugly read out

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#71]
# Description
```
Please provide a short description of what you have encountered below.
```
Time processor error output is distorted. It requires fixing.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```

The expected display is:

```
Change detected, rebuilding site.
2023-02-18 07:13:39.667 +0800
Source changed WRITE         "/.../__components.toml"
ERROR 2023/02/18 07:13:42 
On Page     : /en/blogs/2019-...-io/
File        : en/blogs/2019-...-io/_index.html
Macro       : not applicable
══════════════════════
Time.Created: time: incompatible format '$v'.
%!!(MISSING)(EXTRA string=February 27, 2019 GMT+0800)
══════════════════════
```

# Current Behavior
```
Please specify the current behavior of software below.
```

The current display is:

```
Change detected, rebuilding site.
2023-02-18 07:13:39.667 +0800
Source changed WRITE         "/.../__components.toml"
ERROR 2023/02/18 07:13:42 
On Page     : /en/blogs/2019-...-io/
File        : en/blogs/2019-...-io/_index.html
Macro       : not applicable
══════════════════════
Time.Created: time: incompatible format '$v'.
%!!(MISSING)(EXTRA string=February 27, 2019 GMT+0800)══════════════════════
```

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

1. Create a page.
2. Intentionally set a bad `Created` or `Published` date into the page's `__page.toml` data file.
3. Observe hugo server log.

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
fixed in42f4f729d0975d8e923eb0dc3ece212de9392a85

**Comment by hollowaykeanho** [comment]
Published.

**Comment by hollowaykeanho** [comment]
Released in `v1.1.0`.
